File: Java/08-Laço While/Tabuada.java

/**/

import java.util.Scanner;

class Tabuada {

    public static void main(String[] args) {

        int fator1;
        int contador;
        int produto;

        Scanner input = new Scanner(System.in);

        System.out.println("----------------------------------------");
        System.out.println("            Tabuada Digital             ");
        System.out.println("----------------------------------------");
        System.out.println("Digite o numero que deseja ver a tabuada");
        fator1 = input.nextInt();

        contador = 1;

        // Tabuada de um número até 10
        while (contador <= 10) {

            produto = fator1 * contador;
            System.out.println(fator1 + " x " + contador + " = " + produto);
            contador++;

        }

        System.out.println("----------------------------------------");
        System.out.println("  Powered By Quantum Comp IT Solutions  ");
        System.out.println("----------------------------------------");

        input.close();
    }
}

File: C/10-Laço While/Tabuada.c
#include<stdio.h>

int main(void)
{
    //Tabuada com laço while
    int fator;
    int contador = 0;
    int produto;

    printf("Tabuada \n");
    printf("Digite um numero : ");
    scanf("%d",&fator);

    printf("=-=-=-=-=-=-=-=-=-=-=-=\n");

    while(contador <= 10) {
        produto = fator * contador;
        printf("%d x %d = %d \n",fator, contador, produto);
        contador++;
    }
    return 0;
}
